docs/docs/coverage/language/java.md

```diff
@@ -16,6 +17,7 @@ The following table provides an outline of the features Trivy offers.
 | JAR/WAR/PAR/EAR  |     Trivy Java DB     |     Include      |                  -                   |    -     |
 | pom.xml          | Maven repository [^1] |     Exclude      |                  ✓                   |  ✓[^7]   |
 | *gradle.lockfile |           -           |     Exclude      |                  ✓                   |    ✓     |
+| *.sbt.lock       |          -            |     Exclude      |                  ✓                   |    ✓     |
```

Comment:
The code base doesn't seem to support the dependency graph. It needs which package depends on which package. It returns nil here.
https://github.com/aquasecurity/trivy/blob/5bb0789e1bbc44d4961542f87de4a1cd52d1fe9f/pkg/dependency/parser/sbt/lockfile/parse.go#L65